src/elm/Main.elm

```diff
@@ -433,7 +434,7 @@ type Msg
     | DisableRepo Repository
     | ChownRepo Repository
     | RepairRepo Repository
-    | UpdateRepoEvent Org Repo Field Bool
+    | UpdateRepoEvent Org Repo Repository Field Bool
```

Comment:
that can get confusing :D might have to come up with better naming, but can be a separate PR